src/pubsub/ua_pubsub_networkmessage.c

```diff
@@ -707,6 +772,15 @@ UA_NetworkMessage_calcSizeBinary(UA_NetworkMessage *p, UA_NetworkMessageOffsetBu
     }
 
     if(p->publisherIdEnabled) {
+        if(offsetBuffer){
+            if(offsetBuffer->RTsubscriberEnabled) {
```

Comment:
Just cosmetical, I would preffer a single if statement: offsetBuffer && offsetBuffer->RTsubscriberEnabled. Same type of statement occures several times.

Reply:
Modified.